File: TunnelProject/Opc/OpcGroupDataChangeEventArgs.cs
namespace TunnelProject.Opc
{
    using System;
    using System.Collections.Generic;
    using TunnelProject.Opc.Interfaces;

   
    
        public class OpcGroupDataChangeEventArgs : System.EventArgs
        {
            public List<IOpcObject> ListOfOpcObjects { get; }

            public OpcGroupDataChangeEventArgs(List<IOpcObject> listOfOPCObjects)
            {
                ListOfOpcObjects = listOfOPCObjects;
            }
        }
    }

File: TSA_Rail_System_TunnelProject/Wdw_Sust.OpcDa.Core/eventargs/OPCGroupDataChangeEventArgs.vb
Imports Wdw_Sust.OpcDa.Core.Interfaces

Namespace EventArgs

    Public Class OpcGroupDataChangeEventArgs
        Inherits System.EventArgs

        Private _listOfOPCObjects As List(Of IOpcObject)

        Public Sub New(listOfOPCObjects As List(Of IOpcObject))
            Me._listOfOPCObjects = listOfOPCObjects
        End Sub

        Public ReadOnly Property ListOfOpcObjects() As List(Of IOpcObject)
            Get
                Return _listOfOPCObjects
            End Get
        End Property

    End Class

End Namespace